### PR TITLE
Removed HTML from messages

### PR DESCRIPTION
**Pull request by dannymmr**
Please see:
https://discuss.erpnext.com/t/questions-about-translate-erpnext-com/6739

### CODE REVIEW

File: erpnext/accounts/doctype/cost_center/cost_center.json [hunk 1]
```diff
@@ -66,7 +66,7 @@
    "precision": ""
   }, 
   {
-   "description": "Define Budget for this Cost Center. To set budget action, see <a href=\"#!List/Company\">Company Master</a>", 
+   "description": "Define Budget for this Cost Center. To set budget action, see \"Company Master\"",
```
Comment (anandpdoshi) [inline erpnext/accounts/doctype/cost_center/cost_center.json]: Can you also replace "Company Master" with "Company List"

File: erpnext/setup/doctype/features_setup/features_setup.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
    "permlevel": 0
   }, 
   {
-   "description": "To track items in sales and purchase documents with batch nos<br><b>Preferred Industry: Chemicals etc</b>", 
+   "description": "To track items in sales and purchase documents with batch nos.\"Preferred Industry: Chemicals etc\"",
```
Comment (anandpdoshi) [inline erpnext/setup/doctype/features_setup/features_setup.json]: A spaces after `batch nos.`

**Comment by anandpdoshi** [comment]
Thanks @tmimori :+1:

**Comment by tmimori** [comment]
Updated! Please confirm this.

**Pull request merged**